+nosnoc/+discrete_time_problem/Stewart.m

```diff
@@ -908,8 +919,140 @@ function create_solver(obj, solver_options, plugin)
                     end
                 end
             end
+            
+            if ~isempty(active_set)
+                [IG,IH,~] = obj.process_active_set(active_set);
+            else
+                IG = [];
+                IH = [];
+            end
+            stats = solve@vdx.problems.Mpcc(obj, IG=IG, IH=IH);
+        end
+
+        function [IG,IH,I00] = process_active_set(obj, active_set)
+        % This method takes a nosnoc active set for a PSS and produces an active set for the
+        % complementarity constraints in this problem. It returns these as a boolean array.
+        %
+        % TODO(@anton) this assumes no user complementarities, but how do we handle those?
```

Comment:
We should maybe throw an error message (if it is easy to implement) as we do not support them.
Otherwise, we will implement it only if some relevant example really requires it.